File: android/KidQuest/app/src/main/java/se/kidquest/app/network/ApiConfig.kt
package se.kidquest.app.network

object ApiConfig {
    // Prod-backend (Railway)
    const val BASE_URL: String = "https://backend-production-5c57.up.railway.app/api/v1/"
    // Lokal backend via docker (adventures-utveckling): "http://10.0.2.2:8080/api/v1/"
    // 10.0.2.2 = värddatorn sett från Android-emulatorn.
}

File: ios/KidQuest/KidQuest/ApiConfig.swift
import Foundation

enum ApiConfig {
    // Samma backend som Android-appen använder
    static let baseURL = URL(string: "https://backend-production-5c57.up.railway.app/api/v1/")!
    // Lokal backend via docker (adventures-utveckling):
    // Simulatorn når värddatorn på localhost; ATS undantar localhost så http går bra.
    // static let baseURL = URL(string: "http://localhost:8080/api/v1/")!
}
